plug-ins/groups/group_curative.cpp

```diff
@@ -128,20 +128,25 @@ VOID_SPELL(RemoveCurse)::run( Character *ch, Object *obj, int sn, int level )
 {
     if (IS_OBJ_STAT(obj,ITEM_NODROP) || IS_OBJ_STAT(obj,ITEM_NOREMOVE))
     {
-        if (!IS_OBJ_STAT(obj,ITEM_NOUNCURSE)
-        &&  !savesDispel(level + 2,obj->level,0))
+        if (!IS_OBJ_STAT(obj,ITEM_NOUNCURSE))
         {
-            REMOVE_BIT(obj->extra_flags,ITEM_NODROP);
-            REMOVE_BIT(obj->extra_flags,ITEM_NOREMOVE);
-            act("$o1 загорается голубым светом.", ch,obj,0,TO_ALL);
-            return;
+            if (!savesDispel(level + 2,obj->level,0)) {
+                REMOVE_BIT(obj->extra_flags,ITEM_NODROP);
+                REMOVE_BIT(obj->extra_flags,ITEM_NOREMOVE);
+                obj->getRoom()->echo( POS_RESTING, "%1$^O1 загора%1$nется|ются {1{Cголубым{2 светом!", obj );
+                return;
+            }
+            else {
+                ch->pecho("Проклятье, висящее на %O6, сопротивляется твоим молитвам.", obj);
+                act("", ch,obj,0,TO_CHAR);
```

Comment:
@fiorine ^ extra line.
I'll wait for you to finish working on 'remove curse' messaging below.

Reply:
@ruffinakoza Fixed.